workbench-core/authentication/src/plugins/cognitoUserManagementPlugin.ts

```diff
@@ -164,11 +164,11 @@ export class CognitoUserManagementPlugin implements UserManagementPlugin {
    * @throws {@link InvalidParameterError} if the email parameter is not in a valid format
    * @throws {@link InvalidParameterError} if the email parameter is already in use by a different account
    */
-  public async updateUser(uid: string, user: User): Promise<void> {
+  public async updateUser(id: string, user: User): Promise<void> {
```

Comment:
if we're making breaking changes should we be extracting these inputs into Request and Response objects so we can prevent other breaking changes

Reply:
While these changes are breaking (changing `uid` to `id` in the User interface), they are localized to the AuthN package since SWB doesnt make direct reference to the User interface. Changing UserManagementService to use Request/Response objects is outside the scope of this ticket and will be handled in another task

Reply:
sounds good, this might be something we consider doing earlier because it will be a breaking change across our integration tests.